certificate-policy.md

```diff
@@ -110,7 +108,7 @@ The primary goal of these Requirements is to enable efficient and secure electro
 
 
 ### 1.4.2 Prohibited Certificate Uses
-All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption. 
+All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption. Trusted Role certificates MAY be issued from either the Root or Subordinate CA.
```

Comment:
I fixed this in the Mozilla PR. Some CA products issue certificates to interact with the product. People or devices which may support CA operations such as a server auth cert to a repository or directory or a people cert to the person accessing the CA.